src/pages/index.js

```diff
@@ -20,9 +20,13 @@ import greenPatternImage from '../images/1x/pattern-green.png';
 import meetupsImage from '../images/1x/Meetups.png';
 import buildImage from '../images/1x/Build.png';
 import GradientUnderline from '../components/dev-hub/gradient-underline';
+import BACKGROUND_IMAGE from '../images/1x/homepage-background.png';
```

Comment:
Should we use all caps for these imports as a standard?

Reply:
why

Reply:
Just curious if we wanted anything special for images

Reply:
made it lower case